ideascube/serveradmin/backup.py

```diff
@@ -75,12 +75,37 @@ def format(self):
 
     def save(self):
         """Make a backup of the server data."""
-        return shutil.make_archive(
-            base_name=os.path.join(Backup.ROOT, self.basename),  # W/o ext.
-            format=self.format,
-            root_dir=settings.BACKUPED_ROOT,
-            base_dir='./'
-        )
+        base_name = os.path.join(Backup.ROOT, self.basename)
+        base_name = "{}{}".format(base_name,
+                                  self.FORMAT_TO_EXTENSION[self.format])
+        try:
+            if self.format == 'zip':
```

Comment:
I guess we cannot exclude links from while using `shutil.make_archive`.
And thus I wonder if it's not an occasion to drop `zip` support for backup? Trying to keep things simple.